metron-deployment/roles/ambari_config/vars/single_node_vm.yml

```diff
@@ -87,6 +87,11 @@ configurations:
       supervisor.slots.ports: "[6700, 6701, 6702, 6703, 6704, 6705]"
       storm.local.dir: '{{ storm_local_dir }}'
       topology.classpath: '{{ topology_classpath }}'
+      storm.cluster.metrics.consumer.register: '[{"class": "org.apache.storm.metric.LoggingMetricsConsumer"}]'
```

Comment:
Could you add a comment or explanation about why the changes to the Storm config?  The other changes are pretty straightforward, but it's not obvious why we need config changes (unless you already know its to deal with Storm issues around metrics and killing JMX from the childopts).

Reply:
Sounds good, did that